sqlx-core/src/query_builder.rs

```diff
@@ -139,6 +139,25 @@ where
     /// [`.push_bind()`][Separated::push_bind] methods which push `separator` to the query
     /// before their normal behavior. [`.push_unseparated()`][Separated::push_unseparated] is also
     /// provided to push a SQL fragment without the separator.
+    ///
+    /// ```rust
```

Comment:
```suggestion
    /// ```rust
    /// # #[cfg(feature = "mysql")] {
```